client_code/MultiSelectDropDown/form_template.yaml

```diff
@@ -11,6 +11,7 @@ properties:
 - {name: enable_select_all, type: boolean, default_value: false, group: interaction,
   important: false}
 - {name: visible, type: boolean, default_value: true, group: appearance, important: true}
+- {name: width, type: string, default_value: '', group: appearance}
```

Comment:
If we go with `"fit"` and `"auto"` as possible width values these should be documented

Reply:
I have added fit-content in the next commit as the default_value. Do I need to write out a proper description, or is that enough documentation?